docs/gen_user_guide.py

```diff
@@ -0,0 +1,17 @@
+"""Generate the User Guide."""
+import subprocess
+from pathlib import Path
+
+subprocess.run(
+    [
+        "pandoc",
+        "-f",
+        "markdown",
+        "-t",
+        "html5",
+        "-o",
+        "user_guide.html",
+        f"{Path(__file__).parent}/user_guide.md",
+    ],
```

Comment:
I think we should probably ensure the user guide is written to the docs folder as well, so if we do something like the following we can ensure it's always written there, regardless of where `gen_user_guide.py` is run from.

```suggestion
docs_dir = Path(__file__).parent
subprocess.run(
    [
        "pandoc",
        "-f",
        "markdown",
        "-t",
        "html5",
        "-o",
        f"{docs_dir}/user_guide.html",
        f"{docs_dir}/user_guide.md",
    ],
```

Reply:
Cool looks good, I'll commit that change